File: lista python/at13.py
class Matematica:

    @staticmethod
    def fatorial(n):
        if n == 0 or n == 1:
            return 1
        else:
            return n * Matematica.fatorial(n - 1)


print(f"Fatorial de 7: {Matematica.fatorial(7)}")
print(f"Fatorial de 2: {Matematica.fatorial(2)}")

File: lista java/at13.java
public class Matematica {

    
    public static int fatorial(int n) {
        if (n == 0 || n == 1) {
            return 1;
        } else {
            return n * fatorial(n - 1);
        }
    }

    public static void main(String[] args) {
        
        System.out.println("Fatorial de 8: " + Matematica.fatorial(8));
        System.out.println("Fatorial de 3: " + Matematica.fatorial(3));
    }
}
